content/20.results.md

```diff
@@ -41,14 +41,16 @@ We refer to this dataset as Wiki2019 (available online in [`annotated_names.tsv`
 
 | Group | Training Size | Example Names |
 | ------ | ------ | -------------------- |
-| A | 280,644 | Julie S. Miller, Jesse A. Livezey, Jeremy C Simpson, Chris Smith, Thomas M Drudge |
-| B | 188,918 | Sven Poths, Céline Feillet, Frederik Otzen Bagger, Lars I. Leichert, Sebastian MB Nijman |
-| C | 54,197 | Jee-Hyub Kim, Yoriko Takahashi, Xiaohua Xu, Xuehai Zhang, Yoshihiro Noguchi |
-| D | 66,391 | Beatriz Peñalver Bernabé, Diego Miranda-Saavedra, Marcelo Lobosco, Euler Guimarães Horta, Edgar E Vallejo-Clemente |
-| E | 20,025 | Mahender Kumar Singh, Vidhu Choudhary, Suraj Pradhan, Ramakant Sharma, Vinod Menon |
-| F | 30,703 | Mohammad R. K. Mofrad, Fikret Ercal, Mehdi Yousfi Monod, Ghazaleh Taherzadeh, Noora Al Muftah |
-| G | 4,549 | Tal Vider-Shalit, Itsik Pe'er, Michal Lavidor, Yoav Gothilf, Dvir Netanely |
-| H | 16,105 | Samuel A Assefa, Nyaradzo M. Mgodi, Stanley Kimbung Mbandi, Oyebode J Oyeyemi, Ezekiel Adebiyi |
+| European | 154,890 | Milano Koenders, Gerhard Eck, Eleonora Abbagnato, Alexei Yefimenko, Gérard Gili |
```

Comment:
I just updated the example names here: https://github.com/greenelab/iscb-diversity/blob/master/data/names/example_name_origin.tsv

I'm happy to go with yours but I wanted to make sure we're doing the same thing: honoree/author names that had at least 90% prediction probability in that group.

Also, please note that we made a slight change in the group names. I'd like to spell out specifically that our classification tries to get at the language rather than region (hence, e.g., Hebrew names).

Reply:
Right! I had forgotten that we were using author names highly classified to one group or another, not examples from the training data.